### PR TITLE
[TASK] Set NODE_ENV to production

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-577

### CODE REVIEW

**Comment by wilsonianb** [comment]
:+1:

File: ansible/hosts [hunk 1]
```diff
@@ -11,7 +11,9 @@ api-staging.validators.ripple.com
 [staging:vars]
 ansible_ssh_private_key_file=skynet-keys/ap-southeast-1.pem
 deploy_environment=staging
+node_env=development
```
Comment (stevenzeiler) [inline ansible/hosts]: shouldn't we set node_env=production in staging also?

Comment (stevenzeiler) [inline ansible/hosts]: That way our staging environment is identical to production, less surprises.

Comment (wilsonianb) [inline ansible/hosts]: I was wondering that. Let's do it

Comment (wilsonianb) [inline ansible/hosts]: done

**Comment by stevenzeiler** [comment]
:+1:

**Pull request merged**